deploy/template/nginx_site.conf

```diff
@@ -16,6 +16,17 @@ server {
         }
     }
 
+    location /auth {
```

Comment:
Hm. All URLs in the API doc should be relative to the base path '/api/0'. Any reason that this isn't?